recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -348,6 +352,64 @@ do_newaccount() {
     return $val
 }
 
+do_login() {
+    login_state=$(awk '/common-auth/ { if (substr($1,1,1) ~ /^[# ]/ ) print 1; else print 0}' /etc/pam.d/${2})
+    password_text="Password"
+    google_text="Connagtive Authentication"
+    if [ ${login_state} -eq 0 ]; then
+        password_text="$password_text is active yet"
+    else
+        google_text="$google_text is active yet"
+    fi
```

Comment:
i change to "Static Password" and One-time Password (using Connagtive IoT Device Suite), because the user generates the password on the platform. The Google Authenticator is the program for the realization in the background.